_layouts/tutorial.html

```diff
@@ -9,6 +9,11 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="description" content="Get started developing Apache Kafka® applications with this step-by-step tutorial to learn how to {{ site.data.tutorials[page.static_data].meta-description }} using {{ site.data.stacks[page.stack].pretty }}.">
+  {% if site.data.tutorials[page.static_data].canonical %}
+    {% unless page.stack == 'confluent' %}
```

Comment:
One question - don't we always want a canonical link? Or is it because when `page.stack == confluent` there's a single version (CCloud in this case)